docs/source/configuration/advanced_configuration.md

```diff
@@ -236,10 +236,27 @@ data:
 
 Since both of the file names (`parameters.yml` and `parameters_globals.yml`) match the config pattern for parameters, the `OmegaConfigLoader` will load the files and resolve the placeholders correctly.
 
-```{note}
-Templating currently only works for parameter files, but not for catalog files.
+From Kedro `0.18.10` templating also works for catalog files. To enable templating in the catalog you need to ensure that the template values are within the catalog files or the name of the file that contains the template values follows the same config pattern specified for catalogs.
```

Comment:
I think it'd be nice to separate this section into subheadings like "Parameters" and "Catalog" for readability. Also, mentioning that template values do not undergo duplication checking here as well.

Reply:
Done!